server/controllers/UserController.js

```diff
@@ -252,29 +202,23 @@ class UserController {
     const { email } = req.body.user;
     return User.findOne({
       where: { email },
-    })
-      .then((user) => {
-        if (!user) {
-          return res.status(404)
-            .json({ errors: { message: 'User not found!' } });
-        }
-        const token = tokenService
-          .generateToken({ id: user.id, username: user.username }, 60 * 30);
-        let url = `${process.env.BASEURL}`;
-        url += `/api/users/reset-password?token=${token}`;
-        const mailOptions = emailService.mailOptions(
-          user.email,
-          "Authors Heaven's account token",
-          emailTemplates.resetPasswordTemplate(url, user),
-        );
-        emailService.sendMail(mailOptions)
-          .then(() => res.status(200).json({
-            message: 'Check your email for password reset token',
-          }))
-          .catch(next);
-        return null;
-      })
-      .catch(next);
+    }).then((user) => {
+      if (!user) {
+        errorHelper.throwError('User not found', 404);
+      }
+      const token = tokenService
+        .generateToken({ id: user.id, username: user.username }, 60 * 30);
```

Comment:
You can pass `1800` instead of `60 * 30` as value since it's a constant so that the processor doesn't perform additional arithmetic operations.